jplag.endToEndTesting/src/main/java/model/TestCaseModel.java

```diff
@@ -0,0 +1,44 @@
+package model;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import de.jplag.JPlagComparison;
+import de.jplag.JPlagResult;
+import de.jplag.options.JPlagOptions;
+import de.jplag.options.LanguageOption;
+
+public class TestCaseModel {
```

Comment:
JDoc for all public types and members! Please do it for every file.

Reply:
done